.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on: push
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php-versions: ['7.1', '7.2', '7.3', '7.4', '8.0']
```

Comment:
Didn't we say we were going to exclude 8.0 for now?

Reply:
Yes, this was before that discussion